vclib/src/commonMain/kotlin/at/asitplus/wallet/lib/agent/Validator.kt

```diff
@@ -442,3 +437,12 @@ class Validator(
     }
 
 }
+
+private fun JsonWebKey.equalsCryptographically(second: JsonWebKey) =
```

Comment:
That should be part of kmp-crypto imho

Reply:
See https://github.com/a-sit-plus/kmp-crypto/pull/96